wp-content/themes/blank/footer.php

```diff
@@ -14,16 +14,31 @@
 	</div><!-- #content -->
 
 	<footer id="colophon" class="site-footer">
-		<div class="container">
-			No background image
-			<div>One</div>
-			<div>Two</div>
-			<div>Three</div>
-			<div>Four</div>
+		<div class="container top-foooter">
```

Comment:
footer is sectioned so i believe it can contain a header

```suggestion
		<header class="container top-foooter">
```